include/leds.h

```diff
@@ -28,7 +29,23 @@
 
 extern const std::vector<Pixel> pixels;
 
-void configureAnimations(AnimationStation *as);
-AnimationHotkey animationHotkeys(MPG *gamepad);
+void configureAnimations();
+AnimationHotkey animationHotkeys(MPGS *gamepad);
```

Comment:
Should pass pointer to MPG instead of MPGS. LEDs with no EEPROM/storage is weird, but should be supported.